test/uk/gov/hmcts/contino/GradleBuilderTest.groovy

```diff
@@ -103,11 +103,12 @@ class GradleBuilderTest extends Specification {
   def "runProviderVerification triggers a gradlew hook"() {
     setup:
       def version = "v3r510n"
+      def publishResults = false
     when:
-      builder.runProviderVerification(PACT_BROKER_URL, version)
+      builder.runProviderVerification(PACT_BROKER_URL, version, publishResults)
     then:
       1 * steps.sh({it.startsWith(GRADLE_CMD) &&
-                    it.contains("-Dpact.broker.url=${PACT_BROKER_URL} -Dpact.provider.version=${version} -Dpact.verifier.publishResults=true runAndPublishProviderPactVerification")})
+                    it.contains("-Dpact.broker.url=${PACT_BROKER_URL} -Dpact.provider.version=${version} -Dpact.verifier.publishResults=${publishResults} runProviderPactVerification")})
```

Comment:
This is a breaking change but seems like it wasn't in use, I've discussed offline with Nikola and if anyone is using it he'll help them migrate.

![image](https://user-images.githubusercontent.com/21194782/62051177-bb1a7080-b20a-11e9-9c29-b8375c7651b5.png)